helpers/notification.js

```diff
@@ -0,0 +1,244 @@
+import db from '../models';
+import passportHelper from './passport';
+import mailSender from './utils/mail-sender';
+
+const {
+  User, Follows, Notification
+} = db;
+
+/**
+ * @author Samuel Niyitanga
+ * @exports NotificationHelper
+ * @class NotificationHelper
+ * @description Notification Helper
+ * */
+class NotificationHelper {
+  /**
+     * Set notification option
+     * @param {object} req - an object
+     * @return {object} Returns response
+     * @static
+     */
+  static async setOption(req) {
+    const { option } = req.params;
+    const user = await User.findOne({ where: { id: req.user.id } });
+    const notifsArray = user.dataValues.notification;
+    notifsArray.push(option);
+    const optionSet = await user.update({ notification: notifsArray });
+    return optionSet;
+  }
+
+  /**
+     * Unset notification option
+     * @param {object} req - an object
+     * @return {object} Returns response
+     * @static
+     */
+  static async UnsetOption(req) {
```

Comment:
Why do you pass all the request into the function? Can't you pass only the needed parameters?

Reply:
I am passing all the request because needed parameters are more than one, I thought I would have specified the parameter if it was like only one. Is there anything wrong with passing all the request?

Reply:
Ok, that is fine